doc/man3/OPENSSL_CTX.pod

```diff
@@ -20,7 +20,10 @@ Applications may allocate their own, but may also use C<NULL> to use
 the internal default context with functions that take a C<OPENSSL_CTX>
 argument.
 
-OPENSSL_CTX_new() creates a new OpenSSL library context.
+OPENSSL_CTX_new() creates a new OpenSSL library context. When a non default
```

Comment:
Can we try to remember to make a line break after each sentence and let the local POD->man builders worry about the exact spacing?

Reply:
I'm confused as to why it makes a difference? Anyway I made the requested change.

Reply:
Some have a local convention of one space after a sentence ending in man pages, others two.
That only works (or so experience shows) with sentences are followed by a line break, otherwise it's left to the author, and considering we've never agreed fully within the team, it's better if we leave the decision to local conventions.